### PR TITLE
[test_gap] Need 4 vlans for dhcp relay/dhcpv6 relay

### PR DESCRIPTION
**Pull request by ZhaohuiS**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Need new test: DHCP Relay #3599

#### How did you do it?
Add four vlans config in related t0 topo xml files.
dhcp relay service has only 3 times restart limit, for 4 vlan config, it should restart 4 times for test_dhcp_relay_start_with_uplinks_down case, so add 'systemctl reset-failed dhcp_relay' to avoid the restart  failure.
#### How did you verify/test it?
Deploy updated t0 topo, run test_dhcp_relay or test_dhcpv6_relay to verify.

### CODE REVIEW

File: ansible/roles/test/files/ptftests/dhcpv6_relay_test.py [hunk 1]
```diff
@@ -259,7 +259,7 @@ def verify_relayed_solicit_relay_forward(self):
         masked_packet.set_do_not_care_scapy(scapy.layers.dhcp6.DHCP6_RelayForward, "linkaddr")
 
         # Count the number of these packets received on the ports connected to our leaves
-        solicit_count = testutils.count_matched_packets_all_ports(self, masked_packet, self.server_port_indices)
+        solicit_count = testutils.count_matched_packets_all_ports(self, masked_packet, self.server_port_indices, timeout=4.0)
```
Comment (wangxin) [inline ansible/roles/test/files/ptftests/dhcpv6_relay_test.py:262]: Probably the `qlen` argument for PTF also need to be increased. Example:
https://github.com/Azure/sonic-mgmt/blob/master/tests/fib/test_fib.py#L282

**Review by wangxin — CHANGES_REQUESTED** [review]

File: ansible/vars/topo_t0.yml [hunk 2]
```diff
@@ -73,6 +73,31 @@ topology:
...
+        Vlan2000:
+          id: 2000
+          intfs: [7, 8, 9, 10, 11, 12]
+          prefix: 192.168.4.1/22
+          prefix_v6: fc02:401::1/64
+          tag: 2000
+        Vlan3000:
+          id: 3000
+          intfs: [13, 14, 15, 16, 17, 18]
+          prefix: 192.168.12.1/22
```
[the remark this answers is not in the document]
Comment (ZhaohuiS) [reply ansible/vars/topo_t0.yml]: Done.

File: tests/dhcp_relay/test_dhcpv6_relay.py [hunk 3]
```diff
@@ -244,7 +244,8 @@ def test_dhcp_relay_start_with_uplinks_down(ptfhost, duthosts, rand_one_dut_host
         time.sleep(20)
 
         # Restart DHCP relay service on DUT
-        duthost.shell('systemctl restart dhcp_relay.service')
+        cmds = ['systemctl reset-failed dhcp_relay', 'systemctl restart dhcp_relay']
```
[the remark this answers is not in the document]
Comment (ZhaohuiS) [reply tests/dhcp_relay/test_dhcpv6_relay.py:249]: Done

File: tests/dhcp_relay/test_dhcp_relay.py [hunk 4]
```diff
@@ -351,7 +351,8 @@ def test_dhcp_relay_start_with_uplinks_down(ptfhost, dut_dhcp_relay_data, valida
                       "Not all uplinks go down")
 
         # Restart DHCP relay service on DUT
-        duthost.shell('systemctl restart dhcp_relay.service')
+        cmds = ['systemctl reset-failed dhcp_relay', 'systemctl restart dhcp_relay']
```
[the remark this answers is not in the document]
Comment (ZhaohuiS) [reply tests/dhcp_relay/test_dhcp_relay.py:356]: Done

File: ansible/roles/test/files/ptftests/dhcpv6_relay_test.py (hunk 1 above)
Comment (ZhaohuiS) [reply ansible/roles/test/files/ptftests/dhcpv6_relay_test.py:262]: It didn't failed for dhcp relay test cases during PR test running, I think it just happened on my low performance kvm testbed. Not bring in this change.

**Review by wangxin — APPROVED** [review]

**Pull request merged**